src/transaction/transaction_master_group_complete_manager.cpp

```diff
@@ -77,29 +78,45 @@ namespace cubtx
   //
   void master_group_complete_manager::notify_stream_ack (const cubstream::stream_position stream_pos)
   {
-    /* TODO - disable it temporary since it is not tested */
-    return;
-
     /* TODO - consider quorum. Consider multiple calls of same thread. */
     if (stream_pos >= m_latest_closed_group_end_stream_position)
       {
 	cubthread::entry *thread_p = &cubthread::get_entry ();
 	do_complete (thread_p);
+	er_log_debug (ARG_FILE_LINE, "master_group_complete_manage::notify_stream_ack pos=%llu\n", stream_pos);
       }
   }
 
+  //
+  // get_manager_type - get manager type.
+  //
+  int master_group_complete_manager::get_manager_type () const
+  {
+    return LOG_TRAN_COMPLETE_MANAGER_MASTER_NODE;
+  }
+
   //
   // on_register_transaction - on register transaction specific to master node.
```

Comment:
please add in comment that this function is called while m_group_mutex is latched.

Reply:
Is not needed in case of do_complete -> notify_group_complete.